book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  def add_rental(date, person)
+    @rental.push(date, person, self)
+  end
```

Comment:
- [ ] This method should return a new instance of the `Rental` class the same way your `add_rental` method in the `Person` class returns a new instance of the `Rental` class. Kindly change what this method does. 👍